inscriptions/ISic000553.xml

```diff
@@ -190,30 +195,26 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138312" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-                    Memoriae
-                    <lb n="2"/>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>arci</ex>
-                    </expan>
-                    Aebuti Vernae<lb n="3"/>rarissimi<lb n="4"/>exempli amici
-                    <lb n="5"/>
-                    <expan>
-                        <abbr>T</abbr>
-                        <ex>itus</ex>
-                    </expan>
-                    Aelius Vernae<lb n="6"/>amicus.
-                </ab>
-                <!--This EDR text 138312 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
+                    <lb n="1"/>Memoriae
+                    <lb n="2"/><name><expan><abbr>M</abbr><ex>arci</ex></expan></name> <g ref="#interpunct">·</g> <name>Aebuti</name> <g ref="#interpunct">·</g> <name>Vernae</name>
+                    <lb n="3"/>rarissimi
+                    <lb n="4"/>exempli <g ref="#interpunct">·</g> amici
+                    <lb n="5"/><name><expan><abbr>T</abbr><ex>itus</ex></expan></name> <g ref="#interpunct">·</g> <name>Aelius</name> <g ref="#interpunct">·</g> <name>Vernae</name>
+                    <lb n="6"/>amicus.
+              </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of CIL</note></app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#GC">
+                <p>In memory of Marcus Aebutus Verna, an extraordinary example of a friend, Titus Aelius Verna, friend (dedicated).</p>
```

Comment:
This probably works, but then we have to conclude that 'vernae' was written in error for 'verna' the second time. it is, I imagine, just possible that this is 'friend of Verna' in the final lines. But 'Verna' as cognomen is already rare, and Mommsen suggested that 'vernae amicus' was a corrupt reading for something like 'Vernaculus'. I would leave it as it is for now (the Latin, that is) and go with 'Titus Aeilius, friend of Verna (dedicated this)'